ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-release-pipeline.yml

```diff
@@ -126,37 +126,32 @@ spec:
           workspace: pyxis-ssl-credentials
 
     # Publish image to OCP registry
-    - name: imagestream-manipulation
+    - name: publish-to-ocp-registry
       runAfter:
         - operator-validation
         - get-vendor-related-data
       taskRef:
-        name: imagestream-manipulation
+        name: publish-to-ocp-registry
       params:
         - name: cert_project_id
           value: "$(tasks.certification-project-check.results.certification_project_id)"
         - name: vendor_label
           value: "$(tasks.get-vendor-related-data.results.vendor_label)"
         - name: repository_name
           value: "$(tasks.get-cert-project-related-data.results.repo_name)"
-        - name: isv_pid
-          value: "$(tasks.get-cert-project-related-data.results.isv_pid)"
         - name: dist_method
           value: "$(tasks.get-cert-project-related-data.results.dist_method)"
-        # params necessary to pull the image
-        - name: registry
-          value: "$(params.registry)"
-        - name: image_namespace
-          value: "$(params.image_namespace)"
-        - name: package_name
-          value: "$(tasks.operator-validation.results.package_name)"
+        - name: image
+          value: "$(params.registry)/$(params.image_namespace)/$(tasks.operator-validation.results.package_name)-index:$(tasks.operator-validation.results.bundle_version)"
```

Comment:
I think it would be good if we also store this value for future use, wdyt?

Reply:
No need to store it, if we don't need it yet. It would be like creating a variable because it could be needed in the future- it is not much of an effort to do it in the future, and if we'll add it (pointlessly) now, it can waste the time of confused people who will try to understand why it exists now. 
https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it

Reply:
Agreeing with you. Thanks for the clarification!